internal/util/kubebuilder/project.go

```diff
@@ -50,3 +53,17 @@ func ReadConfig() (*config.Config, error) {
 	}
 	return c, nil
 }
+
```

Comment:
Hi @estroz and @joelanford, 

Regardless of the decision that will be made over we keep or not the operator's name in the PROJECT file, this implementation is now centralized here and this would be the only point that could be changed in the future. So, IMO it is not a blocker for we move forward with this PR.